qemu/tests/cfg/uefishell.cfg

```diff
@@ -0,0 +1,18 @@
+- uefishell:
+    only q35
+    only ovmf
+    type = uefishell
+    start_vm = no
+    boot_order = dcn
+    boot_once = d
```

Comment:
You don't need to specify boot_order and boot_once, because you specify booindex=0 for uefishell.iso. Guest will boot from uefishell.iso.